### PR TITLE
Outputs JSON format only for now

### PR DESCRIPTION
**Pull request by edpin**
issue#6 reports that even JSON formatting wasn't being
output by the CLI in the 'summary' set of commands.
This fixes it to output only JSON format.
Adds a test to validate the formatting of some other options.
A better test is to verify that the values are appearing, without
bothering with checking the precise format (it makes these tests
brittle, since the library might change formatting slightly from version
to version).

### CODE REVIEW

File: zanshincli/main.py [hunk 1]
```diff
@@ -787,18 +789,16 @@ def summary_alert(organization_id: UUID = typer.Argument(..., help="UUID of the
...
 @summary_app.command(name='alert_following')
 def summary_alert_following(organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
                             following_ids: Optional[List[UUID]] = typer.Option(None, help="Only summarize alerts from the specified following, defaults to all.")):
     """
     Gets a summary of the current state of alerts for followed organizations.
     """
     client = Client(profile=global_options['profile'])
-
-    output_iterable(client.get_following_alert_summaries(organization_id, following_ids), empty=0)
+    dump_json(client.get_following_alert_summaries(organization_id, following_ids), empty=0)
```
Comment (rfranco) [inline zanshincli/main.py:801]: Acho que o `empty=0` não existe como parametro no `dump_json`, esta certo?

File: zanshincli/main.py [hunk 2]
```diff
@@ -809,8 +809,7 @@ def summary_scan(organization_id: UUID = typer.Argument(..., help="UUID of the o
     Returns summaries of scan results over a period of time, summarizing number of alerts that changed states.
     """
     client = Client(profile=global_options['profile'])
-
-    output_iterable(client.get_scan_summaries(organization_id, scan_target_ids, days), empty=0)
+    dump_json(client.get_scan_summaries(organization_id, scan_target_ids, days), empty=0)
```
Comment (rfranco) [inline zanshincli/main.py:812]: Acho que o empty=0 não existe como parametro no dump_json, esta

File: zanshincli/main.py [hunk 3]
```diff
@@ -821,8 +820,7 @@ def summary_scan_following(organization_id: UUID = typer.Argument(..., help="UUI
     Returns summaries of scan results over a period of time, summarizing number of alerts that changed states.
     """
     client = Client(profile=global_options['profile'])
-
-    output_iterable(client.get_scan_summaries(organization_id, following_ids, days), empty=0)
+    dump_json(client.get_scan_summaries(organization_id, following_ids, days), empty=0)
```
Comment (rfranco) [inline zanshincli/main.py:823]: Acho que o empty=0 não existe como parametro no dump_json, esta

**Review by rfranco — CHANGES_REQUESTED** [review]

File: zanshincli/main.py (hunk 1 above)
Comment (edpin) [reply zanshincli/main.py]: Ah, good catch! My bad. I made a mistake just prior to sending this out. I've now run pylint and fixed this and other errors. Thanks!

File: zanshincli/main.py (hunk 2 above)
Comment (edpin) [reply zanshincli/main.py]: Ditto.

File: zanshincli/main.py (hunk 3 above)
Comment (edpin) [reply zanshincli/main.py]: Ditto.

**Review by edpin — COMMENTED** [review]
Please take another look. Thanks!

**Review by rfranco — APPROVED** [review]

**Pull request merged**